src/App.jsx

```diff
@@ -14,70 +16,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [thisGood, setGood] = useState('Jam');
+  const NoSelected = 'No goods selected';
+  const clearGoodsSDtate = () => {
+    setGood(null);
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {thisGood ? `${thisGood} is selected` : NoSelected}
+        {thisGood !== null && (
+          <button
+            onClick={() => {
+              setGood('');
+              clearGoodsSDtate();
+            }}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        )
+        }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <>
```

Comment:
```suggestion

```
There is no need to use Fragment if you have a common parent

Reply:
Still not fixed from the previous review